backend/config/initializers/cloudtasker.rb

```diff
@@ -53,6 +53,7 @@
   # Defaults to :development unless CLOUDTASKER_ENV or RAILS_ENV or RACK_ENV is set to something else.
   #
   # config.mode = Rails.env.production? || Rails.env.my_other_env? ? :production : :development
+  config.mode = :production
```

Comment:
it should probably be:
`config.mode = Rails.env.production? ? :production : :development`
so that it continues working using local redis in development

or it should read from RAILS_ENV, but I no longer know if we have that available